internal/transformations/escape_seq_decode.go

```diff
@@ -29,84 +29,81 @@ func doEscapeSeqDecode(input string, pos int) (string, bool) {
 
 	for i < inputLen {
 		if (input[i] == '\\') && (i+1 < inputLen) {
-			c := int8(-1)
+			var c byte
+			var ok bool
```

Comment:
what about  instead of using ok we use default? it is the same. you can exit parent loop using a label

Reply:
I like this one